### ISSUE TITLE
C++ extension stuck in old locale

### ISSUE THREAD

**Issue by digitarald** [microsoft/vscode-cpptools#7665]
Bug type: General

**Describe the bug**
- OS and Version: macOS
- VS Code Version: ersion: 1.57.0-insider, Commit: bf84ee619e0b67443d0307c04348fa575bc3c363
- C/C++ Extension Version: v1.4.1

The C++ extension still shows notifications in Spanish, while the rest of VS Code is in English

**To Reproduce**
*Please include code sample and `task.json` files.*
1. I had switched my OS language to Spanish for testing
2. Switched back to English, many weeks ago

**Expected behavior**
Pick up current VS Code locale.

**Screenshots**
![image](https://user-images.githubusercontent.com/8599/121267464-13900f80-c871-11eb-96e7-cf667fdce891.png)

**Additional context**

**Comment by Colengms** [comment]
The C/C++ Extension does not actually the language of strings it displays.  It uses the vscode-nls-dev and vscode-nls libraries, and is supposed to inherit the currently selected language from VS Code.  If we are displaying strings in the wrong language, that would be a bug in VS Code or vscode-nls.  We've opened issues against VS Code about this in the past, such as https://github.com/microsoft/vscode/issues/82791 in which the language becomes inconsistent if the active language pack extension is removed.  That causes VS Code to revert to the default language (as their strings for the selected language have been removed), but VS Code continues to provide the previously selected language to the C/C++ Extension.

@digitarald Had you deleted a language pack extension?  Does the issue get resolved if you manually select a language pack extension?
